### PR TITLE
[BLOCKED] Added tests for provider refresh

### PR DESCRIPTION
**Pull request by gshefer**
{{ pytest: cfme/tests/containers/test_provider_refresh.py --use-provider cm-env2 }}

Added tests for provider refresh

### CODE REVIEW

File: utils/ocp_cli.py [hunk 1]
```diff
@@ -30,6 +34,7 @@ def __init__(self, provider):
         self.log_line_limit = 500
 
     def run_command(self, *args, **kwargs):
+        raise_on_error = (kwargs.pop('raise_on_error') if 'raise_on_error' in kwargs else False)
```
Comment (RonnyPfannschmidt) [inline utils/ocp_cli.py]: you can use `.pop('raise_on_error', False)` instead

**Comment by gshefer** [comment]
Dependency - https://github.com/ManageIQ/wrapanapi/pull/164

**Comment by gshefer** [comment]
Dependency: https://github.com/ManageIQ/integration_tests/pull/5171

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/containers/test_provider_refresh.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        Replicator: Replicator(rc_name, project_name, provider, appliance),
+        Pod: Pod(pod_name, project_name, provider, appliance),
+        Route: Route(service_name, project_name, provider, appliance),
+        Container: Container(container_name, pod_name, appliance),
+        Volume: Volume(persistant_volume_payload['metadata']['name'], provider, appliance)
+    }
+
+
+@pytest.fixture(scope='module')
+def deleted_resources(provider, appliance, created_resources):
```
Comment (jaryn) [inline cfme/tests/containers/test_provider_refresh.py]: I think you can just delete project. OS should delete all the other garbage.

**Comment by gshefer** [comment]
@jaryn this PR is not RFR yet, I'm planning to convert all the command-line calls to REST. Anyway some comments are still relevant and I will take them into account. Thanks :-)

**Comment by gshefer** [comment]
I'm closing this PR since @jawatts is going to work on these tests.
@jawatts you can use this PR as reference and take into account @jaryn comments which can be helpful.

**Pull request closed without merging**